prql-compiler/src/lib.rs

```diff
@@ -8,25 +8,25 @@
 //!            PRQL
 //!
 //!    (parse) │ ▲
-//! pl_of_prql │ │ prql_of_pl
+//! prql_to_pl │ │ pl_to_prql
 //!            │ │
-//!            ▼ │      pl_of_json
+//!            ▼ │      json::to_pl
```

Comment:
Based on the tests below, I think the labels in lines 13 and 17 should be the other way around.

Reply:
Oh shit, you're right